DOCS-334 Clarify constructing resource name (#1244)
viamrobotics/docs: docs/tutorials/services/plan-motion-with-arm-gripper.md

{{% /tab %}}
{{< /tabs >}}

> Once the Motion Service is accessed, some familiar features become available.

Reviewer: ```suggestion
Once you access the Motion Service, some familiar features become available.
```

Reviewer: This sentence is kinda not helpful though unless we clarify which ones. We could just remove this sentence

> Once the Motion Service is accessed, some familiar features become available.

Author: ```suggestion
```

The Motion service has a method that can get the *pose* of a component relative to a [*reference frame*](../../../services/frame-system/).

> In the tutorial where we interacted with an arm component, we used the `GetEndPosition` method to determine the pose of the end effector of `myArm`.

Reviewer: Do you mind adding a link to the previous tutorial here? I think that's the one referenced

Author: There are three links to that tutorial in the two sections before this. Would it not be overkill?

The `GetPose` method provided by the Motion Service serves a similar function to `GetEndPosition`, but allows for querying of pose data with respect to other elements of the robot (such as another component or the robot's fixed "world" frame).

### Get the `ResourceName`

When you use the [arm API](../../../components/arm/#api), you call methods on your arm component itself.
To use the [Motion Service API](../../../services/motion/#api) on an arm, you need to pass an argument of type `ResourceName` to the Motion Service method.

Add the following to the section of your code where you access the arm:

{{< tabs >}}
{{% tab name="Python" %}}

```python {class="line-numbers linkable-line-numbers"}

> my_arm_resource_name_name = Arm.get_resource_name("myArm")

Reviewer 2: ```suggestion
my_arm_resource_name = Arm.get_resource_name("myArm")
```

```

{{% /tab %}}
{{% tab name="Go" %}}

```go {class="line-numbers linkable-line-numbers"}

> myArmResourceNameName := arm.Named("myArm")

Reviewer 2: ```suggestion
myArmResourceName := arm.Named("myArm")
```

{{% /tab %}}
{{< /tabs >}}

The Motion service has a method that can get the *pose* of a component relative to a [*reference frame*](../../../services/frame-system/).
In the tutorial where we interacted with an arm component, we used the `GetEndPosition` method to determine the pose of the end effector of `myArm`.
The `GetPose` method provided by the Motion Service serves a similar function to `GetEndPosition`, but allows for querying of pose data with respect to other elements of the robot (such as another component or the robot's fixed "world" frame).

### Get the `ResourceName`

> When you use the [arm API](../../../components/arm/#api), you call methods on your arm component itself.

Reviewer 2: ```suggestion
When you use the [arm API](../../../components/arm/#api), you control your arm {{< glossary_tooltip term_id="resource" text="resource" >}}.
```
Sentence is a little bit awk--suggesting as I think this would help more to introduce the following sentence re. ResourceName!

Author: ```suggestion
When you use the [arm API](../../../components/arm/#api), you refer to your arm by the `name` you gave it in your component configuration.
```

> To use the [Motion Service API](../../../services/motion/#api) on an arm, you need to pass an argument of type `ResourceName` to the Motion Service method.

Reviewer 2: ```suggestion
To use the [Motion Service API](../../../services/motion/#api) with an arm, you need to pass an argument of type `ResourceName` to the Motion Service method.
```
(opt) Phrasing
